contracts/rbn-staking/Redeemer.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {ISeizer} from "../interfaces/ISeizer.sol";
+import {IVotingEscrow} from "../interfaces/IVotingEscrow.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {
+  SafeERC20
+} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract Redeemer {
+  using SafeERC20 for IERC20;
+  using SafeMath for uint256;
+
+  // Multisig
+  address public admin;
+  // Maximum % of rbn staked redeemable. 2 decimals i.e 100 * 10 ** 2 = 100% possible to redeem
+  uint8 public maxRedeemPCT;
+  // Seizer implementation
+  ISeizer public seizerImplementation;
+  // veCRV escrow contract
+  IVotingEscrow public votingEscrowContract;
+
+  event RBNRedeemed(uint256 amountRedeemed);
+
+  constructor(address _admin, uint8 _maxRedeemPCT) {
+    require(_admin != address(0), "!_admin");
```

Comment:
One feedback, can we please use longer error messages for the contract? We had to use really short ones for the vault because we were constantly running the bytecode size limit

Reply:
👍